src/LaravelCommon.php

```diff
@@ -2,12 +2,24 @@
 
 namespace LiveIntent\LaravelCommon;
 
+use Illuminate\Support\Facades\App;
 use Illuminate\Support\Facades\Queue;
 use Illuminate\Support\Facades\Route;
+use Illuminate\Auth\AuthServiceProvider;
 use Illuminate\Foundation\Http\Events\RequestHandled;
+use LiveIntent\LaravelCommon\Providers\LIAuthServiceProvider;
 
 class LaravelCommon
 {
+    /**
+     * Register Tessellate token authentication guard
+     */
+    public static function authGuard()
+    {
+        App::register(AuthServiceProvider::class);
```

Comment:
Can you explain why we'd prefer to register the Illuminate\Auth\AuthServiceProvider here rather than leaving the default/standard approach as is?

Reply:
Sure, so I was trying to determine where we want to draw the line between adding a one function call to support adding a default auth to any service versus calling a function and still following convention by adding the dependency in the `Kernel.php` file.